src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -120,42 +122,50 @@ class ResourceService(
-    } yield resource.copy(accessPolicies=policies.toSet)
+    } yield resource.copy(accessPolicies = policies.toSet)
 
-  private def validateCreateResource(resourceType: ResourceType, resourceId: ResourceId, policies: Set[ValidatableAccessPolicy], authDomain: Set[WorkbenchGroupName], userId: WorkbenchUserId, samRequestContext: SamRequestContext): IO[Seq[ErrorReport]] =
+  private def validateCreateResource(resourceType: ResourceType, resourceId: ResourceId, policies: Set[ValidatableAccessPolicy], authDomain: Set[WorkbenchGroupName], userId: WorkbenchUserId, parentOpt: Option[FullyQualifiedResourceId], samRequestContext: SamRequestContext): IO[Seq[ErrorReport]] =
     for {
       resourceIdErrors <- IO.pure(validateUrlSafe(resourceId.value))
-      ownerPolicyErrors <- IO.pure(validateOwnerPolicyExists(resourceType, policies))
+      ownerPolicyErrors <- IO.pure(validateOwnerPolicyExists(resourceType, policies, parentOpt))
       policyErrors <- IO.pure(policies.flatMap(policy => validatePolicy(resourceType, policy)))
       authDomainErrors <- validateAuthDomain(resourceType, authDomain, userId, samRequestContext)
     } yield (resourceIdErrors ++ ownerPolicyErrors ++ policyErrors ++ authDomainErrors).toSeq
 
   private val validUrlSafePattern = "[-a-zA-Z0-9._~%]+".r
+
   private def validateUrlSafe(input: String): Option[ErrorReport] = {
-    if(! validUrlSafePattern.pattern.matcher(input).matches) {
+    if (!validUrlSafePattern.pattern.matcher(input).matches) {
       Option(ErrorReport(s"Invalid input: $input. Valid characters are alphanumeric characters, periods, tildes, percents, underscores, and dashes. Try url encoding."))
     } else {
       None
     }
   }
 
-  private def validateOwnerPolicyExists(resourceType: ResourceType, policies: Set[ValidatableAccessPolicy]): Option[ErrorReport] =
-    policies.exists { policy =>
-      policy.roles.contains(resourceType.ownerRoleName) && policy.emailsToSubjects.nonEmpty
-    } match {
-      case true => None
-      case false =>
-        Option(ErrorReport(s"Cannot create resource without at least 1 policy with ${resourceType.ownerRoleName.value} role and non-empty membership"))
+  private def validateOwnerPolicyExists(resourceType: ResourceType, policies: Set[ValidatableAccessPolicy], parentOpt: Option[FullyQualifiedResourceId]): Option[ErrorReport] =
+    parentOpt match {
+      case None =>
+        // make sure there is an owner policy if there is no parent
+        policies.exists { policy =>
+          policy.roles.contains(resourceType.ownerRoleName) && policy.emailsToSubjects.nonEmpty
+        } match {
+          case true => None
+          case false =>
+            Option(ErrorReport(s"Cannot create resource without at least 1 policy with ${resourceType.ownerRoleName.value} role and non-empty membership"))
+        }
+
+      case Some(_) => None // if a parent exists, the parent's owners are effectively owners of this resource
```

Comment:
And we're sure that there's no way that someone could empty out the members of the Owner policy on the parent?

Reply:
just as sure as no one could empty out the Owner policy on the child